OrdinalV2 prebuild entity type not supported

## Tool
Name: Ludown
Version: 1.3.2
OS: Windows 10

## Describe the bug
If I have a .lu file which contains an OrdinalV2 entity type, the _ludown parse toluis_ command fails.

## To Reproduce
1. Run the _ludown parse toluis_ command on this .lu file
```text
$PREBUILT: ordinalV2

# Intent
- test
```
2. The output will be
```text
[ERROR] Invalid entity definition for $PREBUILT: ordinalV2
Stopping further processing. 
```

## Expected behavior
Parser needs to support ordinalV2 prebuilt entity type.

--- a/packages/Ludown/docs/lu-file-format.md
+++ b/packages/Ludown/docs/lu-file-format.md
@@ -50,7 +50,7 @@ Entity in .lu file is denoted using {\<entityName\>=\<labelled value\>} notation
 ```
 
 LUDown tool supports the following [LUIS entity types](https://docs.microsoft.com/en-us/azure/cognitive-services/LUIS/luis-concept-entity-types)
-- Prebuilt ("datetimeV2", "age", "dimension", "email", "money", "number", "ordinal", "percentage", "phoneNumber","temperature", "url", "datetime", "keyPhrase")
+- Prebuilt ("age", "datetimeV2", "dimension", "email", "geographyV2", "keyPhrase", "money", "number", "ordinal", "ordinalV2", "percentage", "personName", "phoneNumber", "temperature", "url", "datetime")
 - List
 - Simple
 - RegEx

--- a/packages/Ludown/lib/enums/luisbuiltintypes.js
+++ b/packages/Ludown/lib/enums/luisbuiltintypes.js
@@ -4,75 +4,81 @@
  */
 module.exports = {
     consolidatedList: [
-        "datetimeV2",
         "age",
+        "datetimeV2",
         "dimension",
         "email",
+        "geographyV2",
+        "keyPhrase",
         "money",
         "number",
         "ordinal",
+        "ordinalV2",
         "percentage",
+        "personName",
         "phoneNumber",
         "temperature",
         "url",
-        "datetime",
-        "keyPhrase",
-        "geographyV2",
-        "personName"
+        "datetime"
     ],
     perLocaleAvailability: {
         "de-de": {
             "datetime": "datetimeV2",
             "geographyV2": null,
-            "personName": null
+            "personName": null,
+            "ordinalV2": null
         },
         "zh-cn": {
             "datetime": "datetimeV2",
             "keyPhrase": null,
-            "geographyV2": null
+            "geographyV2": null,
+            "ordinalV2": null
         },
         "en-us": {
             "datetime": "datetimeV2"
         },
         "fr-fr": {
-            "datetime": "datetimeV2"
+            "datetime": "datetimeV2",
+            "geographyV2": null,
+            "ordinalV2": null,
+            "personName": null
         },
         "fr-ca": {
             "datetime": "datetimeV2",
             "geographyV2": null,
+            "personName": null,
+            "ordinalV2": null,
             "personName": null
         },
         "es-es": {
             "datetime": "datetimeV2",
             "geographyV2": null,
-            "personName": null
+            "personName": null,
+            "ordinalV2": null
         },
         "es-mx": {
             "datetimeV2": "datetime",
-            "age": null,
-            "dimension": null,
-            "money": null,
-            "ordinal": null,
-            "percentage": null,
-            "temperature": null,
-            "datetime": null,
             "geographyV2": null,
+            "ordinalV2": null,
             "personName": null
         },
         "it-it": {
             "datetimeV2": "datetime",
             "geographyV2": null,
-            "personName": null
+            "personName": null,
+            "ordinalV2": null
         },
         "ja-jp": {
             "datetimeV2": "datetime",
             "geographyV2": null,
-            "personName": null
+            "personName": null,
+            "ordinalV2": null
         },
         "pt-br": {
             "datetime": "datetimeV2",
             "geographyV2": null,
-            "personName": null
+            "personName": null,
+            "ordinalV2": null
         },
         "ko-kr": {
             "datetimeV2": null,
@@ -85,13 +91,33 @@ module.exports = {
             "temperature": null,
             "datetime": null,
             "geographyV2": null,
-            "personName": null
+            "personName": null,
+            "ordinalV2": null
         },
         "nl-nl": {
             "geographyV2": null,
             "personName": null,
             "datetimeV2": null,
+            "datetime": null,
+            "ordinalV2": null
+        },
+        "tr-tr": {
+            "age": null,
+            "datetimeV2": null,
+            "dimension": null,
+            "email": null,
+            "geographyV2": null,
+            "keyPhrase": null,
+            "money": null,
+            "number": null,
+            "ordinal": null,
+            "ordinalV2": null,
+            "percentage": null,
+            "personName": null,
+            "phoneNumber": null,
+            "temperature": null,
+            "url": null,
             "datetime": null
         }
     }
-}
+};

--- a/packages/Ludown/lib/ludown-parse-ToLuis.js
+++ b/packages/Ludown/lib/ludown-parse-ToLuis.js
@@ -36,7 +36,7 @@ if (process.argv.length < 4) {
     if (program.luis_culture) {
         // List of supported LUIS.ai locales. 
         // From: https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-language-support
-        const LUISLocales = ['en-us', 'fr-ca', 'zh-cn', 'nl-nl', 'fr-fr', 'de-de', 'it-it', 'ja-jp', 'ko-kr', 'pt-br', 'es-es', 'es-mx'];
+        const LUISLocales = ['en-us', 'fr-ca', 'zh-cn', 'nl-nl', 'fr-fr', 'de-de', 'it-it', 'ja-jp', 'ko-kr', 'pt-br', 'es-es', 'es-mx', 'tr-tr'];
         if (!(LUISLocales.includes(program.luis_culture.toLowerCase()))) {
             process.stderr.write(chalk.default.yellowBright(`\nWARN: Unrecognized LUIS locale. Supported locales are - ${LUISLocales.toString()} \n\n`));
         }

